tests/integration/test_list_simulations.py

```diff
@@ -96,3 +98,4 @@ def test_list_simulations(celery_app,
     assert data["page_count"] == 3
     assert len(data["simulations"]) == 1
     assert data["simulations"][0]["start_time"] == start_time_of_newest_simulation
+    sleep(5)
```

Comment:
That cannot be accepted as solution, I know it may fix the issue, but we need to understand the source of the problems